### PR TITLE
twoliter: create initial project with kits

### LINKED ISSUE #116: kits: create an initial test project for local kits
## Description

Create a project that contains a `kits` directory with at least one Kit in it. Create Cargo.toml and other metadata for this kit which defines the kits interface. Create a variant in the project which depends on the kit(s) defined in the `kits` directory.

## Definition of Done

When the above project exists. It is not necessary to implement the kit building code in the same PR as this test project.

### PR DESCRIPTION
**Pull request by ecpullen**
**Issue number:**

Closes #116 

**Description of changes:**

Adds 2 new projects, 1 that bundles Bottlerocket into a `core` kit, the other adds Bottlerocket packages as packages.

**Testing done:**

N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: tests/projects/project2/variants/hello-ootb/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "net.ifnames=0",
+    "netdog.default-interface=eth0:dhcp4,dhcp6?",
+    "quiet",
+]
+
+[lib]
+path = "../variants.rs"
+
+[build-dependencies]
+hello-agent = { path = "../../kits/hello-kit" }
```
Comment (webern) [inline tests/projects/project2/variants/hello-ootb/Cargo.toml]:
```suggestion
hello-kit = { path = "../../kits/hello-kit" }
```

File: tests/projects/project2/variants/hello-ootb/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    # docker
+    "docker-cli",
+    "docker-engine",
+    "docker-init",
+    "docker-proxy",
+    # tools
+    "login",
+    "iputils",
+    "strace",
+    "chrony-tools",
```
Comment (webern) [inline tests/projects/project2/variants/hello-ootb/Cargo.toml]:
```suggestion
    "hello-agent",
    "hello-go",
```

**Review by webern — COMMENTED** [review]
delete project3

**Review by webern — APPROVED** [review]
Can we name the projects in a self-documenting way? I would call this one `local-kit`.

Can you also add a README.md at the top of the `local-kit` project directory that explains that the purpose of the test project is to introduce a simple local kit project with no external kit dependencies?

**Review by webern — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]

**Pull request merged**